exercises/bank-account/canonical-data.json

```
@@ -0,0 +1,317 @@
+{
+  "exercise": "bank-account",
+  "comments": [
+    "This exercise is a good candidate to practice concurrency, which",
+    "may not be available or possible in the implementing language track.",
+    "In this case, all tests can be executed on a single thread. If there",
+    "are tests that don't work using a single thread, these should be",
+    "marked with a concurrency scenario.",
+    "----------------------------------------------------------------",
+    "For tests which requires multiple expected values/errors",
+    "Should put the expected values/errors at each operation",
```

```suggestion
    "Should put the expected values/errors as a field of each operation",
```

"at" didn't seem clear to me - is this suggestion better?